ProcessMaker/processmaker
# FOUR-20192 the cases list does not show the correct status when the case is ERROR

## Issue & Reproduction Steps
The cases list does not show the correct status when the case is ERROR

## Solution
- Update the case status on the process request error log

https://github.com/user-attachments/assets/b71fe6ae-4025-4a9e-aa83-5e793a219939

## How to Test
1. Have an empty ABE settings
2. Create a process with an ABE element
3. Run the process
4. Once the ABE is reached an exception should be thrown
e.g.
```
PhpImap\Mailbox::__construct(): Argument #1 ($imapPath) must be of type string, null given, called in /packages/package-actions-by-email/src/PluginServiceProvider.php on line 144
```
5. Go to the Cases page
6. The case status must be `ERROR`

## Related Tickets & Packages
- [FOUR-20192](https://processmaker.atlassian.net/browse/FOUR-20192)

## Code Review Checklist


[FOUR-20192]: https://processmaker.atlassian.net/browse/FOUR-20192?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
